data_generation/README.md

```diff
@@ -0,0 +1,37 @@
+# Generate data
+
+Use this scripts to generate data with the Inductiva API.
+
+The first thing that you need to do is export your api key as an
+environment variable:
+
+```bash
+export INDUCTIVA_API_KEY = <Your API Key>
+```
+
+This will prevent accidentally committing it to github since now we
+only need to do:
+
+```
+inductiva.api_key = os.environ['INDUCTIVA_API_KEY']
```

Comment:
Technically, this won't be necessary, since when doing import inductiva this will be done automatically.

Reply:
Is this feature available already?

Reply:
I believe so! But you may need to set it manually on `.bashrc` or respective file for your terminal of use.